### DISCUSSION TITLE
Linking against FFP ROM symbols

### DISCUSSION THREAD

**Discussion by ssav7912** [BartmanAbyss/vscode-amiga-debug#252] [General]
How is LD configured to link against functions stored in ROM? 

When trying to build using FFP functions I get undefined reference linker errors to those functions (e.g. `SPDiv`, `SPMul`), an error I don't get with other ROM functions like `AllocMem`. Is this something I can configure LD to find, or should I just be writing wrappers to call into those functions by their offset on ROM?

I can see that at least some of the symbols are in the elf files in the `/bin/symbols` directory, but I'm not sure if that's relevant or useful for me.

Thanks

**Comment by BartmanAbyss** [comment]
There's no automatic linking for amiga system libraries.
You need to manually load `mathffp.library` to use `SPDiv`, etc.; see here: http://amigadev.elowar.com/read/ADCD_2.1/Libraries_Manual_guide/node047E.html
